routes/controllers/channel.controller.js

```diff
@@ -0,0 +1,29 @@
+const createError = require('http-errors');
+const mongoose = require('mongoose');
+
+const Channel = require('../../models/Channel');
+
+exports.getChannel = async (req, res, next) => {
+  try {
+    const { channelId } = req.params;
+    const channel = Channel.findById(channelId);
```

Comment:
```suggestion
    const channel = await Channel.findById(channelId);
```
await 키워드가 빠져있는 것 같아요